lib/bridge.js

```diff
@@ -239,15 +302,13 @@ Bridge.prototype.addCallback = function addCallback (hostname, port, listen, pat
       .then(function checkResponse (response) {
         if (response.success) {
           return self
-              .getCallbacks()
-              .then(function findCallback (callbacks) {
-                for (var i = 0; i < callbacks.length; i++) {
-                  if (callbacks[i].url === url) {
-                    return new Callback(self, callbacks[i].id, url);
-                  }
-                }
-
-                throw new Error(response);
+              .getCallbacks(true)
+              .then(callbacks => {
+                var callback = callbacks.find(callback => callback.url === url);
+                if (callback)
```

Comment:
Like already commented: "Please add braces for the if. Opening brace at the end of the if-line"
Please look everywhere in your changes